src/RequestExecutor/WsRequestExecutor.ts

```diff
@@ -165,18 +172,15 @@ export class WsRequestExecutor implements RequestExecutor {
   ): Record<string, string | string[]> {
     return Object.entries(headers).reduce(
       (
-        common: Record<string, string | string[]>,
+        result: Record<string, string | string[]>,
         [key, value]: [string, string | string[]]
       ) => {
-        if (
-          !['sec-websocket-version', 'sec-websocket-key'].includes(
-            key.trim().toLowerCase()
-          )
-        ) {
-          common[key] = value;
+        const headerName = key.trim().toLowerCase();
+        if (!WsRequestExecutor.FORBIDDEN_HEADERS.has(headerName)) {
+          result[key] = value;
         }
 
-        return common;
+        return result;
```

Comment:
Just a comment. I don't think refactoring is a good idea in such PRs.

Reply:
I tend to agree with you but I needed to make the list of forbidden headers public, so the refactoring was somewhat inevitable. At least it should be a bit more performant now 😉